services/traefik-forward-auth-mgmt/0.3.9/defaults/cm.yaml

```diff
@@ -18,15 +18,13 @@ data:
       port: 4181
     traefikForwardAuth:
       enabled: true
-      # oidcUri will be overridden by the init-container
+      # oidcUri, clientID, clientSecret will be overridden by the TFAController
       oidcUri: "https://dex.${releaseNamespace}.svc.cluster.local:8080/dex"
-      caSecretName: kommander-traefik-certificate
       clientId: traefik-forward-auth
       clientSecret:
+        value: "placeholder"
```

Comment:
[Question] Why do we add `placeholder` as value? Shouldn't we skip this if we don't have proper value for it?

Reply:
It is analog to the existing definition in `traefik-forward-auth`. I guess it makes it clearer that this is a value that is later overwritten by the controller.